pom.xml

```diff
@@ -486,7 +486,7 @@
       <snapshots>
         <enabled>false</enabled>
       </snapshots>
-      <id>${nexus.release.repository.id}</id>
+      <id>${nexus.release.repository.id}-release</id>
```

Comment:
as there wan't an entry for the release repo entry at all recently, I hope just changing the name here should not cause side effects 😬